implementations/rust/ockam/ockam_command/src/space/list.rs

```diff
@@ -7,9 +7,18 @@ use crate::node::util::delete_embedded_node;
 use crate::space::util::config;
 use crate::util::api::{self, CloudOpts};
 use crate::util::{node_rpc, Rpc};
-use crate::CommandGlobalOpts;
+use crate::{docs, CommandGlobalOpts};
 
+const LONG_ABOUT: &str = include_str!("./static/list/long_about.txt");
+const AFTER_LONG_HELP: &str = include_str!("./static/list/after_long_help.txt");
+
+/// List spaces
 #[derive(Clone, Debug, Args)]
+#[command(
+    arg_required_else_help = true,
```

Comment:
This is likely why the test is failing.

Reply:
True! Didn't see that one, thanks for fixing it 🙏